### PR TITLE
Add type hints to urllib3.contrib._securetransport.low_level

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: src/urllib3/contrib/_securetransport/low_level.py [hunk 1]
```diff
@@ -34,7 +46,7 @@ def _cf_data_from_bytes(bytestring):
     )
 
 
-def _cf_dictionary_from_tuples(tuples):
+def _cf_dictionary_from_tuples(tuples: List[Tuple[Any, ...]]) -> CFDictionary:
```
Comment (sethmlarson) [inline src/urllib3/contrib/_securetransport/low_level.py:49]: Type should be `List[Tuple[Any, Any]]`

File: src/urllib3/contrib/_securetransport/low_level.py [hunk 2]
```diff
@@ -123,10 +135,12 @@ def _cf_string_to_unicode(value):
         string = buffer.value
     if string is not None:
         string = string.decode("utf-8")
-    return string
+    return string  # type: ignore
```
Comment (sethmlarson) [inline src/urllib3/contrib/_securetransport/low_level.py:138]: Can `string` somehow be `None` here?

File: noxfile.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 # ignored.
 TYPED_FILES = {
     "src/urllib3/contrib/__init__.py",
+    "src/urllib3/contrib/_securetransport/low_level.py",
```
Comment (sethmlarson) [inline noxfile.py:12]: Forgot to mention it in the previous PR, can we add the `bindings.py` to this list? Just so we run mypy on it even if it's basically skipped.

**Review by sethmlarson — CHANGES_REQUESTED** [review]
Looks pretty good, few comments then we can merge.

**Comment by hramezani** [comment]
@sethmlarson Thanks for the review.
I addressed your comments related to type hints(force pushed to the first commit)
And added another commit to add `bindings.py` to `TYPED_FILES` list

**Review by sethmlarson — APPROVED** [review]
LGTM :rocket:

**Pull request merged**